translations/code-of-conduct.portuguese_br.md

```diff
@@ -0,0 +1,59 @@
+
+- Usar linguagem acolhedora, inclusiva e de gênero neutro (por exemplo: ao invés de "Hey guys" (Olá caras),
+  você pode utilizar "Hey folks" (Olá pessoal) ou "Hey all" (Olá a todos))
+- Ser respeitoso em relação a diferentes pontos de vista e experiências
+- Aceitar críticas construtivas
+- Focar no que é melhor para a comunidade
+- Mostrar empatia em relação a outros membros da comunidade
+
+Exemplos de comportamento inaceitável pelos participantes inclui:
+
+- O uso de linguagem ou imagens sexualizas além de atenção ou investidas sexuais não desejadas
+- Trollar, comentários com insultos e/ou depreciativos, além de ataques pessoais ou políticos
+- Assédio/Perseguição pública ou privada
+- Publicar dados privados de outros membros da comunidade, como endereços físicos ou eletrônicos, sem permissão explícita
+- Outras condutas que poderiam ser trivialmente consideradas inapropriadas em um ambiente profissional
+
+## Nossas Responsabilidades
+
+Os mantenedores do projeto são responsavéis por clarificar os padrões de comportamento aceitável e é esperado que tomem ações corretivas justas e apropriadas em relação a qualquer instância de comportamento inaceitável.
+
+Os mantenedores do projeto tem o direito e a responsabilidade de remover, editar ou rejeitar comentários, commits, código, alterações na wiki, issues, e outras contribuições que não estejam alinhadas com esse Código de Conduta, além poder banir tempo ou permanentemente qualquer contribuidor por algum comportamente que seja considerado inapropriado, ameaçador, ofensivo ou malicioso.
+
+## Escopo
+
+Esse Código de Conduta aplica-se tanto no espaço desse projeto quanto no espaço público quando um indíviduo estiver representado o projeto ou sua comunidade. Exemplos de representação do projeto ou sua comunidade incluem a utilização de um endereço de e-mail oficial do projeto, postagem em uma mídia social com conta oficial do
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of projeto, ou agindo como um membro escolhido pelo projeto em um evento online ou offline. Representação do projeto pode ser mais definida e clarificada pelos mantenedores do projeto.
+
+## Cumprimento
+
+Instâncias de comportamento abusivo, ameaçador ou qualquer outro comportamento inaceitável devem ser reportadas contatando o time do projeto em community@hasura.io. Todas reclamações serão revisadas e investigadas e resultarão em uma resposta que seja considerada necessária e adequada para as circunstâncias. O time do projeto é obrigado a manter confidencialiade em relação ao relator do incidente.
+Mais detalhes sobre políticas de cumprimento específicas poderão ser postadas em separado.
+
+Mantenedores do projeto que não seguem ou não executam o Código de Conduta em boa fé podem sofrer temporárias ou permanentes repercussões como determinando por outros membros da liderança do projeto.
```

Comment:
Mantenedores do projeto que não seguem ou não executam o Código de Conduta de boa-fé podem sofrer sanções temporárias ou permanentes conforme for determinado pelos líderes do projeto.